src/domain/event/utils.ts

```diff
@@ -13,3 +18,70 @@ export const getEventPlaceholderImage = (id: string): string => {
 
   return EVENT_PLACEHOLDER_IMAGES[index];
 };
+
+/**
+ * Get first availanle language of an event
+ * @param {object} eventData
+ * @return {string}
+ */
+export const getFirstAvailableLanguage = (eventData: EventQuery): Language => {
+  if (eventData.event?.name.en) return SUPPORT_LANGUAGES.EN;
```

Comment:
Is this the correct order? should FI be first?

Reply:
maybe should be same order as in the language buttons?

<img width="712" alt="Screenshot 2020-05-19 at 11 34 39" src="https://user-images.githubusercontent.com/15219142/82304279-cb335480-99c4-11ea-86a1-e4debd3d6426.png">